packages/cojson-storage-sqlite/src/tests/storage.sqlite.test.ts

```diff
@@ -374,7 +377,9 @@ test("should recover from data loss", async () => {
     [
       "client -> CONTENT Group header: true new: After: 0 New: 3",
       "client -> CONTENT Map header: true new: After: 0 New: 1",
-      "client -> CONTENT Map header: false new: After: 1 New: 3",
+      "client -> CONTENT Map header: false new: After: 3 New: 1",
+      "storage -> KNOWN CORRECTION Map sessions: header/4",
+      "storage -> CONTENT Map header: false new: After: 1 New: 3",
```

Comment:
This is expected.

The test checks what happens when the storage loses some transactions

Before: newContentSince was sending all the content since the last knownState of storage
Now: new local transactions are pushed without checking the knownState and corrected in case of errors